packages/tupaia-web-server/src/routes/ExportSurveyResponsesRoute.ts

```diff
@@ -65,6 +69,12 @@ export class ExportSurveyResponsesRoute extends Route<ExportSurveyResponsesReque
       centralQuery,
     );
 
+    if (response.emailTimeoutHit) {
+      throw new Error(
+        'This export is taking a long time! The data will be emailed to you when finished.',
+      );
```

Comment:
This isn't _really_ an error. But it was a quick way to get this running without having to redefine the types for the response from this endpoint, since I don't know how we want to organise the frontend of this functionality